SpiceSharpTest/Models/RLC/CAP/CapacitorTests.cs

```diff
@@ -65,6 +66,95 @@ public void When_Capacitor_LowpassRCTransient_Expect_Reference()
             AnalyzeTransient(tran, ckt, exports, references);
         }
 
+        [Test]
+        public void When_Capacitor_Has_TC1_Zero_TC2_Zero_And_30Degrees_LowpassRCTransient_Expect_Reference()
```

Comment:
This method name is too long. Use the same format as the others test methods.

Reply:
@svenboulanger Changed

Reply:
This project uses convention number 6 from [here](https://dzone.com/articles/7-popular-unit-test-naming). The new names are still not following this convention (the underschore after `Capacitor` should be removed.

Reply:
@svenboulanger Ok. I've changed name again.